### PR TITLE
Fix for r-hat calculation on multidimensional variables

### PR DESCRIPTION
**Pull request by fonnesbeck**
Found a bug in `forestplot` related to calculating R-hat for multidimensional variables. This seems to fix it.

### CODE REVIEW

File: pymc/diagnostics.py [hunk 1]
```diff
@@ -133,18 +133,24 @@ def gelman_rubin(mtrace):
...
-        # Calculate within-chain variance
-        W = np.mean(np.var(x, axis=1, ddof=1))
+            # Estimate of marginal posterior variance
+            Vhat = W*(n - 1)/n + B/n
 
-        # Estimate of marginal posterior variance
-        Vhat = W*(n - 1)/n + B/n
+            return np.sqrt(Vhat/W)
+
+        except ValueError:
```
Comment (jsalvatier) [inline pymc/diagnostics.py:152]: Maybe put a comment here, explaining what this is catching? I don't quite get what problem is being addressed here.

Comment (fonnesbeck) [inline pymc/diagnostics.py:152]: Done below.

**Comment by twiecki** [comment]
Looks good, merging.

**Comment by twiecki** [comment]
Says there are "unmerged commits. " @fonnesbeck ?

**Comment by fonnesbeck** [comment]
Don't know. Merging worked for me, without any warnings.

**Comment by fonnesbeck** [comment]
Pushed my merge just now.

**Pull request closed without merging**